docs/source/geo_def.rst

```diff
@@ -109,23 +183,42 @@ content
    area_extent:
      lower_left_xy: [-5326849.0625, -5326849.0625]
      upper_right_xy: [5326849.0625, 5326849.0625]
+   top_left_extent: [-5326849.0625, 5326849.0625]
```

Comment:
Why are the extents and the area_extent both specified? I don't think this is a realistic use case. The examples listed here should stick with the original 3 or 4 use cases that this PR was originally addressing and maybe a couple extra that show the same/similar use cases but with units specified or a 2-element parameter specified with one element.